pkg/cmd/roachtest/sysbench.go

```diff
@@ -114,18 +122,17 @@ func runSysbench(ctx context.Context, t *test, c *cluster, opts sysbenchOptions)
 func registerSysbench(r *testRegistry) {
 	for w := sysbenchWorkload(0); w < numSysbenchWorkloads; w++ {
 		const n = 3
-		const cpus = 16
+		const cpus = 32
 		opts := sysbenchOptions{
 			workload:     w,
 			duration:     10 * time.Minute,
-			concurrency:  8 * cpus,
-			tables:       4,
-			rowsPerTable: 1000000,
+			concurrency:  4 * cpus,
+			tables:       10,
+			rowsPerTable: 10000000,
```

Comment:
is rowsPerTable something we want in the test name?

Reply:
The table size is only used when populating the dataset before running the benchmark and it doesn't have much of an effect on the benchmark's performance as long as it's sufficiently large. If anything, we'd want to add workload concurrency to the test name. I wasn't going to do that because I wanted to keep this simple,  but I guess it's worth it because we'll eventually want to sweep this benchmark over a series of concurrencies. Done.